File: server/src/main/kotlin/com/pidulgi/server/admin/dto/response/AdminGetReportResponse.kt
package com.pidulgi.server.admin.dto.response

import com.pidulgi.server.report.entity.type.ReportType
import java.time.LocalDateTime

data class AdminGetReportResponse(

    val reportId: Long,
    val type: ReportType,
    val reporterNickname: String,
    val reportedNickname: String,
    val reason: String?,
    val createdAt: LocalDateTime,
)

File: admin/types/AdminGetReportResponse.ts
export type ReportType =
  | "ABUSE"
  | "SPAM"
  | "MINOR"
  | "SEXUAL"
  | "FAKE"
  | "ETC";

export type AdminGetReportResponse = {
  reportId: number;
  type: ReportType;
  reporterNickname: string;
  reportedNickname: string;
  reason: string | null;
  createdAt: string;
};
